mlflow/store/tracking/rest_store.py

```diff
@@ -115,6 +116,7 @@ def create_experiment(self, name, artifact_location=None, tags=None):
         Returns:
```

Comment:
maybe we should edit the `name` docstring to mention that there is character limit now